src/actions/Entities/VaccinePrescriptionActions.js

```diff
@@ -49,11 +51,19 @@ const updateEditing = (value, field) => (dispatch, getState) => {
   dispatch(update(newVaccinePrescriptionId, field, value));
 };
 
+const cancel = () => dispatch => {
+  batch(() => {
+    dispatch(NavigationActions.back());
+    dispatch(reset());
+  });
+};
```

Comment:
At the moment this action isn't getting dispatched when the hardware back button is pressed or the extra cool back button on the top left of the nav bar.

Some options I can think of
1) Add dispatching of the reset action into `NavigationActions.back` - I did this for sensor work, I think it's the worst option.
2) Add a case in the reducer for the action `Navigation/BACK` - this is probably my preference
3) Leave it, it gets sorted when navigating to the page anyway, I think - this is probably the better option

Reply:
good point. I've added in 1fa783de
While I don't like the fact that it resets on every back navigation in the app.. it's better than not 🤷‍♂️